searchform.php

```diff
@@ -10,13 +10,12 @@
 ?>
 
 <form method="get" id="searchform" action="<?php echo esc_url( home_url( '/' ) ); ?>" role="search">
-	<label class="sr-only" for="s"><?php esc_html_e( 'Search', 'uds-wordpress-theme' ); ?></label>
```

Comment:
Careful, this is for screen-readers (accessibility). We need to verify whether this can be safely removed. (and if it has been removed from the example Storybook code, was that approved and confirmed to be ok?) I would ask Steve Ryan for confirmation with Kathy Marks, who has final approval on all accessibility features and markup.